packages/paste-design-tokens/package.json

```diff
@@ -18,9 +18,18 @@
     "build:watch": "gulp dev",
     "clean": "rm -rf ./dist",
     "tokens": "gulp default",
-    "test": "jest"
+    "test": "jest",
+    "tsc": "yarn build"
   },
   "devDependencies": {
+    "@types/gulp": "^4.0.6",
```

Comment:
I pulled these deps out from the monorepo `package.json` file into this package specific one because they're only used as devDeps in this one package.